server/src/server.jl

```diff
@@ -0,0 +1,164 @@
+export
+    start
+
+# map JSON function name to Struct (bmi_service.jl)
```

Comment:
I wouldn't export this generic name. `WflowServer.start()` makes it clear what is started. From julia 1.11+ it would make sense to mark it `public` though.

https://github.com/JuliaLang/julia/blob/master/NEWS.md#new-language-features

```suggestion
# map JSON function name to Struct (bmi_service.jl)
```